src/resdk/resources/sample.py

```diff
@@ -30,10 +30,12 @@ class Sample(SampleUtilsMixin, BaseCollection):
     collection = DictResourceField(
         resource_class_name="Collection", access_type=FieldAccessType.WRITABLE
     )
-    data = QueryRelatedField("Data")
-    relations = QueryRelatedField("Relation")
-    annotations = QueryRelatedField("AnnotationValue")
-    predictions = QueryRelatedField("PredictionValue")
+    data = QueryRelatedField("Data", "entity")
```

Comment:
Why do you need to change this?

Reply:
For filtering... Otherwise entity is renamed to the default endpoint, sample.